### PR TITLE
release 0.12.0

### PR DESCRIPTION
**Pull request by burrbull**

### CODE REVIEW

File: svd-encoder/CHANGELOG.md [hunk 1]
```diff
@@ -5,8 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## [v0.12.0] - 2021-11-11
 
+- Bump `svd-rs`
 - Add `protection` encoding
```
Comment (Emilgardis) [inline svd-encoder/CHANGELOG.md:11]: the `## Unreleased` should stay, only add a new h2 with the date

```
## Unreleased

## [v0.12.0] - 2021-11-11
```

File: svd-parser/CHANGELOG.md [hunk 2]
```diff
@@ -5,8 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## [v0.12.0] - 2021-11-11
```
Comment (Emilgardis) [inline svd-parser/CHANGELOG.md:8]: dito

File: svd-rs/CHANGELOG.md [hunk 3]
```diff
@@ -5,8 +5,9 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
-## Unreleased
+## [v0.12.0] - 2021-11-11
```
Comment (Emilgardis) [inline svd-rs/CHANGELOG.md:8]: dito

**Comment by burrbull** [comment]
fixed

**Review by Emilgardis — APPROVED** [review]
bors r+

**Pull request merged**